src/api/goals.js

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+
+export default (dispatch, action, storage) => {
+  const router = new express.Router();
+
+  router.get('/add/:goal', (request, response) => {
+    const timer = storage.read()[request.timerId];
+
+    if (timer.goals.length >= 5) {
+      return response
+        .status(403)
+        .json({ message: 'Too many goals' })
+        .end();
+    }
+
+    dispatch(action.AddGoal(request.params.goal, request.timerId));
+
+    return response.status(204).end();
+  });
+  router.get('/:goal/complete', (request, response) => {
```

Comment:
I don't know if I like `/text/action` when add/remove use `/action/text`, but maybe that's a separate ticket

Reply:
See #45